app/src/main/res/values/dimens.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="main_result_tv_size">36sp</dimen>
+    <dimen name="main_result_tv_height">100sp</dimen>
```

Comment:
this is not correct, the height of a component will be in DP, only the size of a text goes in SP

Reply:
Dobe